src/pybert/pybert.py

```diff
@@ -362,7 +347,7 @@ def __init__(self, name="(noname)", enabled=False, min_val=0.0, max_val=0.0, val
 
         # Super-class initialization is ABSOLUTELY NECESSARY, in order
         # to get all the Traits/UI machinery setup correctly.
-        super(TxTapTuner, self).__init__()
+        super().__init__()
```

Comment:
Can you tell me what happened here?
Do you know why `TxTapTuner` and `self` were being passed originally?

Reply:
That is no longer required as of python 3.0 but was required when you were running python 2.7!
 https://docs.python.org/3/library/functions.html#super